packages/jest/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@apollo/utils.jest",
```

Comment:
a little meh on the grab-bag package, can this be `@apollo/utils.jest-graphql-ast-serializer`?

Also once this is a thing that's actually exported from somewhere rather than just hidden inside AS it should probably have graphql in its name like above.

Reply:
Also looks like this is missing a peer dep on `graphql`. And the fact that it needs one suggests that `@apollo/utils.jest` is not a good package shape since you can easily imagine Apollo Jest utils that don't depend on graphql

Reply:
[355b5e0](https://github.com/apollographql/apollo-utils/pull/92/commits/355b5e0249be4160d0435f88eee09a59f47e6d8c)